axlearn/common/aot_compilation.py

```diff
@@ -0,0 +1,90 @@
+"""Utilities for ahead-of-time compilation.
+
+Reference:
+https://docs.google.com/document/d/1Y5IdmvAZA7UtMHAWkRh8k2PscVoG5FvMH9-E6hygsyY/
```

Comment:
This Google doc asks me to sign in in order to view it. Is it publicly accessible?

Reply:
It should be. Are you able to access it after signing in?

Reply:
No, it says "you need to request access."

Reply:
Bummer. Changed to https://jax.readthedocs.io/en/latest/aot.html.